cartographer_ros_msgs/srv/WriteState.srv

```diff
@@ -13,5 +13,6 @@
 # limitations under the License.
 
 string filename
+uint32[] trajectory_ids
```

Comment:
I do not like having to specify trajectory ids by default. Perhaps change this to which trajectory IDs to exclude, so the default is "serialize everything"?